impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -70,3 +70,30 @@ service KeyTransparencyService {
     option (google.api.http) = { get: "/v1/domain/info" };
   }
 }
+
+// The KeyTransparencyBackendService API provides batch access to the directory of public keys. 
+service KeyTransparencyBackendService {
+  // BatchGetEntries returns full proofs for a set of user_ids.
+  rpc BatchGetEntries(keytransparency.v1.types.BatchGetEntriesRequest) returns (keytransparency.v1.types.BatchGetEntriesResponse) {
+    option (google.api.http) = { get: "/v1/users" };
+  }
+
+
+  // BatchUpdateEntries will put a set of UpdateEntryRequests into the queue. 
+  rpc BatchUpdateEntries(keytransparency.v1.types.BatchUpdateEntriesRequest) returns (keytransparency.v1.types.BatchUpdateEntriesResponse) {
+    option (google.api.http) = {
+      post: "/v1/users"
+      body: ""
+    };
+  }
+
+  // BatchSetEntries uses an authorized_public key to perform a set request on multiple 
+  // entries at once.  
+  rpc BatchSetEntries(keytransparency.v1.types.BatchSetEntriesRequest) returns (keytransparency.v1.types.BatchSetEntriesResponse) {
```

Comment:
Perhaps I need better names, but the idea is that `BatchUpdate` is just a batch version of `Update`, meaning that it is pretty dumb and just puts things in the mutation table. 

`Set` is really more of a server side client that has been equipped with a server `authorized_key`  Perhaps this needs a better name like `AccountResetService` or something like that.  In order to make integrations with other backend services easier, I thought it would be best to make the API interface for updating accounts as simple as possible.  This in particular is an API that needs to be authorized.  The other two are just higher volume versions of the earlier methods.